src/Renderer/ThreeExtended/Feature2Mesh.js

```diff
@@ -219,13 +221,14 @@ function coordinateToPolygonExtruded(coordinates, properties, options) {
     let offset = 0;
     let offset2 = 0;
     let nbVertices = 0;
-
+    let minAltitude = Infinity;
     /* eslint-disable-next-line */
     for (const id in coordinates.featureVertices) {
         // extract contour coodinates and properties of one feature
         const { contour, property } = extractFeature(coordinates, properties, id);
         // get altitude and extrude amount from properties
         const altitudeBottom = getAltitude(options, property);
+        minAltitude = Math.min(minAltitude, altitudeBottom);
```

Comment:
What is this `minAltitude` stuff? And why is that necessary?

Reply:
It's the minimum altitude of the vertex of the points. It's used to position the object3D on the surface normal